src/config.rs

```diff
@@ -282,17 +280,44 @@ impl Config {
             ResolvAddr::resolve_or_exit,
         );
 
+        let changed_daemon_dir = config.daemon_dir != default_daemon_dir();
+
+        let mut default_cookie_path = config.daemon_dir;
```

Comment:
Oh, wait, if daemon dir affects cookie maybe we should keep using it?

Reply:
it only affects the default cookie location, and that is only used when no cookie and no username and password are set

Reply:
Yes, it could be somewhat useful for people who don't care to figure out the exact location of the cookie. But I could be convinced this is a too weird feature to support.

Reply:
Let's keep `daemon_dir` for now, and only remove `timestamp` in the PR.